content/id/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,477 @@
+yang sesuai dengan *Ingress*, selama *service-service* yang didefinisikan (`s1`, `s2`) ada.
+Apabila *Ingress* selesai dibuat, maka kamu dapat melihat alamat IP dari *loadbalancers* 
+pada kolom `address`.
+
+{{< note >}}
+Kamu mungkin saja membutuhkan konfigurasi default-http-backend [Service](/en/docs/concepts/services-networking/service/) 
+bergantung pada [Ingress controller](/en/docs/concepts/services-networking/ingress-controllers) yang kamu pakai.
+{{< /note >}}
+
+### *Name based virtual hosting*
+
+*Name-based virtual hosts* memungkinkan mekanisme *routing* berdasarkan trafik HTTP ke beberapa *host name* dengan alamat IP yang sama. 
+
+```none
+foo.bar.com --|                 |-> foo.bar.com s1:80
+              | 178.91.123.132  |
+bar.foo.com --|                 |-> bar.foo.com s2:80
+```
+
+Ingress di bawah ini memberikan perintah pada *loadbalancer* untuk melakukan mekanisme *routing* berdasarkan 
+[header host](https://tools.ietf.org/html/rfc7230#section-5.4).
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: name-virtual-host-ingress
+spec:
+  rules:
+  - host: foo.bar.com
+    http:
+      paths:
+      - backend:
+          serviceName: service1
+          servicePort: 80
+  - host: bar.foo.com
+    http:
+      paths:
+      - backend:
+          serviceName: service2
+          servicePort: 80
+```
+
+Jika kamu membuat sebuah Ingress tanpa mendefinisikan *host* apa pun, maka 
+trafik web ke alamat IP dari kontroler Ingress tetap dapat dilakukan tanpa harus 
+menyesuaikan aturan *name based virtual host*. Sebagai contoh, 
+*resource* Ingress di bawah ini akan melakukan pemetaan trafik  
+dari `first.bar.com` ke `service1`, `second.foo.com` ke `service2`, dan trafik lain 
+ke alamat IP tanpa *host name* yang didefinisikan di dalam *request* (yang tidak memiliki *request header*) ke `service3`.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: name-virtual-host-ingress
+spec:
+  rules:
+  - host: first.bar.com
+    http:
+      paths:
+      - backend:
+          serviceName: service1
+          servicePort: 80
+  - host: second.foo.com
+    http:
+      paths:
+      - backend:
+          serviceName: service2
+          servicePort: 80
+  - http:
+      paths:
+      - backend:
+          serviceName: service3
+          servicePort: 80
+```
+
+### TLS
+
+Kamu dapat mengamankan *Ingress* yang kamu punya dengan memberikan spesifikasi [secret](/en/docs/concepts/configuration/secret)
+yang mengandung *private key* dan sertifikat TLS. Saat ini, Ingress hanya 
+memiliki fitur untuk melakukan konfigurasi *single TLS port*, yaitu 443, serta melakukan terminasi TLS. 
+Jika *section* TLS pada Ingress memiliki spesifikasi *host* yang berbeda,
+*rules* yang ada akan dimultipleksikan pada *port* yang sama berdasarkan 
+*hostname* yang dispesifikasin melalui ekstensi TLS SNI. *Secret* TLS harus memiliki 
```

Comment:
typo: dispesifikasin

Reply:
probably give some reference to SNI for folks who dont know

https://en.wikipedia.org/wiki/Server_Name_Indication